pkgs/development/tools/yarn-berry/default.nix

```diff
@@ -18,13 +26,9 @@ stdenv.mkDerivation (finalAttrs: {
     hash = if berryVersion == 4 then hash_4 else hash_3;
   };
 
-  buildInputs = [
-    nodejs
-  ];
+  buildInputs = [ nodejs ];
```

Comment:
This diff shouldn't be necessary with the latest nixfmt. See https://github.com/NixOS/nixfmt/pull/224 .

Reply:
see https://github.com/NixOS/nixpkgs/pull/355053#discussion_r1837038425